### ISSUE TITLE
Add support for WebACL (WAF2 and WAF Classic)

### ISSUE THREAD

**Issue by tonivdv** [getlift/lift#178]
It's sometimes necessary to protect our cloudfront distribution with a
WAF2/WAF Classic web ACL. A very first simple approach would be to allow linking
an existing WAF resource.

I propose something like:

```yaml
constructs:
    website:
        # ...
        webAclId: arn:aws:wafv2:us-east-1:{account}:global/webacl/{webacl_name}/{webacl_id}
```

**Comment by tonivdv** [comment]
This could be solved with fix #191

**Comment by fredericbarthelet** [comment]
@tonivdv we now released Lift with support for [extensions](https://github.com/getlift/lift#extend-lift-constructs). Feel free to try it out and see if this covers your requirements. I'll close this issue in the meantime :) Thanks
